bin/index.js

```diff
@@ -9,13 +9,13 @@ const fs = promisifyAll(require('fs'));
 const program = require('commander')
     .version(pkg.version)
     .description(`${pkg.name} (v${pkg.version}): ${pkg.description}`)
-    .usage('<file>')
-    .arguments('<file>')
+    .usage('<file> [options]')
+    .option('-o, --output', 'write to HTML file')
     .parse(process.argv);
 
 const srcFilename = path.join(program.args[0]);
 
 fs.readFileAsync(srcFilename, 'utf8')
     .then(readme => formatter(readme))
-    .then(html => console.log(html))
-    .catch(err => console.error(err));
\ No newline at end of file
+    .then(html => program.output ? fs.writeFile(program.output, html) : console.log(html))
```

Comment:
There are two issues with `fs.writeFile`:
* it does not support writing to a folder that doesn't exist (yet)
* it does not return a promise

You could use `fs.writeFileAsync` so that you can `.catch` any errors.
For creating missing directories on the fly you can use the `mkdirp` module.

I think you will end up needing something like this: https://github.com/jbmoelker/jinja-compat-tests/blob/master/lib/save-file.js